components-core/src/main/java/com/adyen/checkout/components/core/internal/analytics/AnalyticsPlatformParams.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 27/2/2024.
+ */
+
+package com.adyen.checkout.components.core.internal.analytics
+
+import androidx.annotation.RestrictTo
+import androidx.annotation.VisibleForTesting
+import com.adyen.checkout.components.core.BuildConfig
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+object AnalyticsPlatformParams {
+
+    const val CHANNEL = "android"
+
+    var platform = AnalyticsPlatform.ANDROID.value
```

Comment:
In the previous code we named these `actualPlatform` and `actualVersion` (check `AnalyticsMapper`). Is it intended to change them to `platform` and `version`?

Reply:
Yes, before we used both forms with and without 'actual' prefixed, so it would be confusing if didn't have it. Now this class is just providing them, so it would be confusing when you're consuming this class

Reply:
Ah good point. But the code now looks a bit confusing because `CHANNEL` is different from the rest:
```kotlin
version = AnalyticsPlatformParams.version,
channel = AnalyticsPlatformParams.CHANNEL,
platform = AnalyticsPlatformParams.platform,
```
Can we make all 3 values look the same for an external consumer?